linux/devdoc/socket_transport_linux_requirements.md

```diff
@@ -65,15 +66,15 @@ MOCKABLE_FUNCTION(, SOCKET_TRANSPORT_HANDLE, socket_transport_create, SOCKET_TYP
 
 `socket_transport_create` creates a socket transport.
 
-`socket_transport_create` shall ensure `type` is either `SOCKET_CLIENT`, or `SOCKET_SERVER`.
+**SOCKET_TRANSPORT_LINUX_11_001: [** `socket_transport_create` shall ensure `type` is either `SOCKET_CLIENT`, or `SOCKET_SERVER`. **]**
```

Comment:
I think this spec should normally read "If type is not SOCKET_CLIENT or SOCKET_SERVER, then socket_transport_create shall fail and return NULL" - just so it is clear what will be returned.